lib/foreman/telemetry_sinks/rails_logger_sink.rb

```diff
@@ -0,0 +1,31 @@
+module Foreman::TelemetrySinks
+  class RailsLoggerSink
+    def initialize(opts = {})
+      @level = Logger.const_get(opts[:level].try(:to_sym).try(:upcase) || :debug)
+    end
+
+    def add_counter(name, description, instance_labels)
+      Rails.logger.add(@level, "Registering counter #{name} labels #{instance_labels.inspect}")
```

Comment:
The current logging setup provides the ability to log to a special logger (e.g. [app] or [sql]). I think this could be an excellent use case that when using the log sink to log to a [stats] or [telemetry] or [telem] logger for easier parsing.

Reply:
Good idea, added.